chord_metadata_service/experiments/migrations/0002_auto_20200326_1339.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.2.10 on 2020-03-26 17:39
+
+import chord_metadata_service.restapi.validators
```

Comment:
this migration was generated by django because the JsonSchemaValidator was updated but there are no actual changes to the models. Should I discard it?

Reply:
If you downgrade, get rid of it and then generate new migrations, is it there (based on your description, it shouldn't)?

Reply:
I am just thinking that it can collide with migrations on Romain's branch because he is working with experiments app: it means there will be two migration files that have a dependency to an initial migration, and both files will have 0002 sequence number. So when someone checks out the code and migrates how django knows what migrations apply first? 
that's why maybe just delete this useless migration :)

Reply:
Ah right, tricky situation for sure. In this case sounds like you can simply delete it. But there's also the option of squashing multiple migrations together, if it comes to that.